Mathlib/NumberTheory/NumberField/ProductFormula.lean

```diff
@@ -0,0 +1,108 @@
+/-
+Copyright (c) 2024 Fabrizio Barroero. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Fabrizio Barroero
+-/
+import Mathlib.NumberTheory.NumberField.FinitePlaces
+
+/-!
+# The Product Formula for number fields
+In this file we prove the Product Formula for number fields: for any non-zero element `x` of a
+number field `K`, we have `∏|x|ᵥ=1` where the product runs over the equivalence classes of absoulte
+values of `K` and the `|⬝|ᵥ` are suitably normalized.
+
+## Main Results
+* `NumberField.FinitePlace.prod_eq_inv_abs_norm`: for any non-zero element `x` of a number field
+`K`, the product `∏|x|ᵥ` of the absolute values of `x` associated to the finite places of `K` is
+equal to the inverse of the norm of `x`.
+* `NumberField.prod_abs_eq_one`: for any non-zero element `x` of a number field `K`, we have
+`∏|x|ᵥ=1` where the product runs over the equivalence classes of absoulte values of `K`.
```

Comment:
```suggestion
`∏ |x|ᵥ=1`, where the product runs over the equivalence classes of absolute values of `K`.
```
Spaces for readability. (I would also prefer an empty line below the `#` and `##` lines, but this is a matter of taste.)
Maybe say something about how the absolute value is normalized?